test/test_logger.rb

```diff
@@ -113,6 +113,17 @@ def test_re_stamping_existing_timestamped_file
         assert(task.file != task.current_file)
     end
 
+    def test_no_suffix_log_file
+        task.overwrite_existing_files = false
+        task.auto_timestamp_files = true
+        task.file = "/tmp/rock_logger_test"
+        touch_file = File.new(task.file, "w")
+        assert(!task.has_port?('time'))
+        assert(task.createLoggingPort('time', '/base/Time', []))
+        generate_and_check_logfile
+        assert(task.file != task.current_file)
```

Comment:
You're actually not testing the bug that has been fixed, that is that the timestamp will be added as a suffix.

Reply:
My bad. Should be resolved now..